### PR TITLE
update contributing guide and hacktoberfest info

### PR DESCRIPTION
**Pull request by dcalvin**
This PR:

1. update the contributing guide in the contributor folder
2. remove the contributing guide in the top directory
3. add Hacktoberfest event

### CODE REVIEW

File: Hacktoberfest/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Hacktoberfest 2020
```
Comment (shenli) [inline Hacktoberfest/README.md:1]: https://github.com/kubernetes/community/tree/master/events 
I suggest putting the events in a dedicated dir.

**Review by shenli — COMMENTED** [review]
Rest LGTM

File: Hacktoberfest/README.md (hunk 1 above)
Comment (dcalvin) [reply Hacktoberfest/README.md:1]: Makes sense. However, as we have ongoing activities such as High-Performance challenge program, this change may affect the accessibility of certain related files. How about we adjust globally when the activity is over @AndreMouche also suggested the same to me earlier.

**Review by zz-jason — APPROVED** [review]
LGTM

**Comment by shenli** [comment]
LGTM
Please file an issue about migrate events things.

**Comment by dcalvin** [comment]
> LGTM
> Please file an issue about migrate events things.

Sure

> LGTM
> Please file an issue about migrate events things.

done https://github.com/pingcap/community/issues/309

File: contributors/README.md [hunk 2]
```diff
@@ -1,184 +1,141 @@
 # How to contribute
```
Comment (zz-jason) [inline contributors/README.md:1]: this title can be removed.

**Pull request merged**